install_scripts/templates/kubernetes/migrate.sh

```diff
@@ -278,30 +277,21 @@ startAppOnK8s() {
     restoreSecrets
 
     logSubstep "restore console settings"
-    set +e
     local needsActivation=0
-    /usr/local/bin/replicatedctl migration import < "${TMP_DIR}/migration.json" 2>"${TMP_DIR}/import.txt"
-    if [ "$?" -ne 0 ] ; then
-        if grep -q 'Activation code invalid' "${TMP_DIR}/import.txt" ; then
-            needsActivation=1
-        else
-            cat "${TMP_DIR}/import.txt"
-            exit 1
-        fi
+    if cat "${TMP_DIR}/migration.json" | kubectl exec -i $(kubectl get pods -o=jsonpath="{.items[0].metadata.name}" -l tier=master) -- /bin/sh -c 'replicatedctl migration import || true' 2>&1 | grep -q 'Activation code invalid' ; then
```

Comment:
grep works this way without writing to a file

Reply:
I feel like this would be impossible to troubleshoot though.  Your call really.

Reply:
Yeah, I'll change it back